sdk/logical/identity.pb.go

```diff
@@ -270,31 +275,33 @@ func init() {
 	proto.RegisterMapType((map[string]string)(nil), "logical.Group.MetadataEntry")
 }
 
-func init() { proto.RegisterFile("sdk/logical/identity.proto", fileDescriptor_4a34d35719c603a1) }
+func init() {
+	proto.RegisterFile("sdk/logical/identity.proto", fileDescriptor_4a34d35719c603a1)
+}
```

Comment:
This change is likely due to me having a different protoc version than the last person who generated the protobuf files. My version is:
```
$ protoc --version
libprotoc 3.11.4
```